test/saithrift/src/switch_sai_rpc_server.cpp

```diff
@@ -619,6 +619,36 @@ class switch_sai_rpcHandler : virtual public switch_sai_rpcIf {
       return;
   }
 
+  void sai_thrift_get_vlan_attribute(sai_thrift_attribute_list_t& thrift_attr_list, const sai_thrift_object_id_t vlan_id) {
```

Comment:
It's a bit different between sai calls and saithrift calls. All other APIs behave like that as well. Let's keep consistency